ai_diffusion/util.py

```diff
@@ -296,3 +299,110 @@ def acquire_elements(l: list[QOBJECT]) -> list[QOBJECT]:
         if obj is not None:
             sip.transferback(obj)
     return l
+
+
+# Resave the image with PNG metadata
+def resave_image_with_metadata(img_path: Path, job: "Job"):  # noqa: F821
```

Comment:
I don't think this function or the explanation in comments is needed.

Also if the function only needs the `params` from the job, pass that in directly instead of the entire job.

Reply:
Change to just pass in the JobParams